Admin/Admin.php

```diff
@@ -2926,7 +2926,7 @@ public function getAccessMapping()
     /**
      * {@inheritdoc}
      */
-    public function checkAccess($action, $object = null)
+    public function getAccess()
```

Comment:
Right, there is probably no need to get the role and then check it is in the template. Let's use `hasAccess`.